src/xcode/ENA/ENA/Source/Services/HealthCertificate/Models/__test__/Name+ExtensionTests.swift

```diff
@@ -17,6 +17,15 @@ class Name_ExtensionTests: XCTestCase {
 		XCTAssertEqual(name.groupingStandardizedName, name.standardizedName)
 		XCTAssertEqual(name.groupingStandardizedName, "THOMAS<ARMIN MEYER")
 	}
+	
+	func testGIVEN_ValidName_THEN_GroupingStandardizedNameIsTrimmed() {
+		// GIVEN
+		let name = Name(familyName: "Meyer ", givenName: "Thomas Arming ", standardizedFamilyName: "MEYER<", standardizedGivenName: "THOMAS<ARMIN<"
```

Comment:
```suggestion
		let name = Name(familyName: "Meyer ", givenName: "Thomas Armin ", standardizedFamilyName: "MEYER<", standardizedGivenName: "THOMAS<ARMIN<"
```
It the test working correct?

Reply:
Good catch. But the test is working correct because it it only takes into account standardizedGivenName.